connect_box/__init__.py

```diff
@@ -513,8 +526,17 @@ async def _async_initialize_valid_token(self) -> None:
                 headers=self.headers,
                 timeout=10,
             ) as response:
-                await response.text()
+                html = await response.text()
+                apiversion = re.findall("CommonAPI\.js\?v=(\d*)", html)
                 self.token = response.cookies["sessionToken"].value
+                if apiversion and len(apiversion) > 0:
```

Comment:
I guess that this is sufficient to check if the list is not empty.

```suggestion
                if apiversion:
```

Reply:
Changed as suggested